resource/mwcwallet_style.css

```diff
@@ -659,4 +659,17 @@ QTabBar::tab:hover{
 
 QTabBar::tab:selected{
     font-weight:bold;
+}
+
+QGroupBox {
+    font: bold;
```

Comment:
For light theme white color was needed. Please check if  light theme works fine
    color: white;